packages/beacon-node/src/network/gossip/interface.ts

```diff
@@ -122,6 +122,21 @@ export type GossipModules = {
   chain: IBeaconChain;
 };
 
+export type GossipBeaconNode = {
```

Comment:
shouldn't this rather be an `interface`?

Reply:
Definitely could be an interface. I don't know that we have clear project-wide opinions on the use of one vs the either.

Reply:
IMO if it is implemented by a class within the project directly or even outside in an external library I would use `interface` otherwise should just use `type`

Reply:
Yeah, `type` here is a bit sloppy. Can clean it up in a future PR.
I'm continuing to do some work to clean this up, aiming to move the network to a worker, so will continue touching these pieces of code.